src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function handleClick() {
+  const promise1 = new Promise((resolve) => {
+    resolve('Promise was resolved!');
+  });
+
+  promise1.then(handleSuccess).catch(handleError);
+}
+
+function createRejectingPromise(message) {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(new Error(message));
+    }, 3000);
+  });
+}
+
+logo.addEventListener('click', handleClick);
+
+const promise2 = createRejectingPromise('Promise was rejected!');
+const messageDiv = document.createElement('div');
+
+document.body.appendChild(messageDiv);
+
+const handleSuccess = (message) => {
+  messageDiv.className = 'message';
+  messageDiv.textContent = message;
+};
+
+const handleError = (errorMessage) => {
+  messageDiv.className = 'message error-message';
+  messageDiv.textContent = errorMessage;
+};
+
+promise2.then(handleSuccess).catch((error) => handleError(error.message));
```

Comment:
The catch method should just take the handleError function as an argument, without wrapping it in another function. This is because the catch method will automatically pass the error as an argument to the function you provide.